### DISCUSSION TITLE
IIASA IAM framework

### DISCUSSION THREAD

**Discussion by zsalimian** [iiasa/message_ix#548] [MESSAGE modeling] [answered]
Dear Sir,

I would like to use IIASA IAM framework which consists of GLOBIOM,MAGICC,....

I would like to know if different elements of IIASA IAM framework such as  GLOBIOM,MAGICC and GAINS are open source like MESSAGE-ix? Can I install and apply them like MESSAGE-ix?
Are there any tutorial sample for other models in IIASA IAM framework?

**Comment by LauWien** [comment]
Dear @zsalimian,
Since this is neither about (a) a bug or error, nor (b) a requested feature in the software per se, I have transferred and converted this from an issue of the message-ix-models repository to a “Discussion” in the message_ix repository.

**Comment by LauWien** [comment, marked as answer]
Dear @zsalimian, 
Thank you for reaching out and your interest!
Following the status of the three models you mentioned:
-  GLOBIOM: There will be a MESSAGEix-GLOBIOM global model release in the near future, however as of today it is not open-source.
- MAGICC: There are ongoing developments towards releasing this in the future, but as of today it is not open-source.
- GAINS: Please have a look at the online model [here](http://gains.iiasa.ac.at/models/gains_models4.html).
